packages/amplify-appsync-simulator/tsconfig.json

```diff
@@ -5,6 +5,7 @@
         "sourceMap": true,
         "outDir": "lib",
         "lib": [
+            "dom",
```

Comment:
Why DOM?

Reply:
It was required for something during build, sorry I can't recall what was it, but it was specific ;-)

Reply:
https://github.com/aws-amplify/amplify-cli/pull/1893#discussion_r310374979 😜